python/cuml/fil/fil.pyx

```diff
@@ -210,7 +211,7 @@ cdef class ForestInference_impl():
                              ' to the documentation')
         return storage_type_dict[storage_type_str]
 
-    def predict(self, X, preds=None):
+    def predict(self, X, predict_proba=False, preds=None):
```

Comment:
This method is not called with the default values, so feel free to remove them.

Reply:
we decided to address the cuml/ensemble/randomforestregressor.pyx and cuml/ensemble/randomforestclassifier.pyx question separately, keeping the default values since it's called directly from there.